editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -1279,7 +1279,7 @@ export function SpecialSizeMeasurementsKeepDeepEquality(): KeepDeepEqualityCall<
       hasPositionOffsetEquals &&
       textDirectionEquals &&
       hasTransformEquals &&
-      borderRadiusEquals
+      borderRadiusEquals.areEqual
```

Comment:
This will be superseded by #2930, so I'll deal with that merge conflict before merging this.